### DISCUSSION TITLE
MySQL insert produces error & Other questions about BB

### DISCUSSION THREAD

**Discussion by sarper57** [Budibase/budibase#8339] [Help]
There are 11 columns in the data file and trying to insert some records using 6 columns after retrieving via REST API.
Below is a screenshot of executing query with error. However if I use the PHPMyAdmin with same code, MySQL inserts the records without any error.
I am hoping someone knows how to get around it.
![2022-10-19](https://user-images.githubusercontent.com/42462205/196980467-c2cd9d8a-4efd-4e32-97cf-41be77c38853.png)

After multiselecting rows and using actions executing an update query followed by refreshing the data provider selected rows haven't been deselected. Is tis normal or I need to write a script to empty the selected rows of the table and then refresh?

My last question relates to **action If Continue/Stop** - If it stops , there is no alternative to inform the user why  it had stopped as it will not process rest of the actions, is this a short sight of BB?

thank you in advance for your helps

**Comment by agruenberg** [comment]
Can you post the entire default value for your binding? It is cut off in the screen shot.

**Reply by agruenberg** [reply]
Maybe this will help #8326

**Reply by sarper57** [reply]
Hi
the default value was **123,123,null,'wrong import Values',null,null** , I have also tried with **'123','123','Invalid Import Values','NA',0,''**
also tried with 11 column values none worked.
